### DISCUSSION TITLE
delegate library support not built-in warning with Magick++

### DISCUSSION THREAD

**Discussion by jackthenewbie** [ImageMagick/ImageMagick#5943] [Help] [answered]
### ImageMagick version

7

### Operating system, version and so on

ubuntu-20

### Description

``delegate library support not built-in 'AvantGarde-Book' (Freetype) @ warning/annotate.c/RenderFreetype/2067.``
As I read on doc, cmd line tool should have font path and stuff. However with Magick++.h install (without cmd line) doesn't work, warning above happens when I run my program with ``./main logo.png AvantGarde-Book``
```c++
InitializeMagick(*argv);
  Image image;
  try { 
    // Read a file into image object 
    cout<<"Reading file: "<<argv[1]<<endl;
    image.read(argv[1]);
    image.font(argv[2]);
    image.annotate("Hello World", EastGravity);\
    image.write("result.png");
  } 
```
I think I may miss something in the docs, im kindda confuse right now, please help.

**Comment by dlemstra** [comment, marked as answer]
Your version of ImageMagick is build without Freetype support.

**Comment by jackthenewbie** [comment]
my distro doesn't have ``libfreetype6-dev``, thanks dlemstra
